tests/pyfunc/test_spark.py

```diff
@@ -1014,3 +1014,24 @@ def predict(self, context, model_input, params=None):
         )
         res = good_data.withColumn("res", udf("a")).select("res").toPandas()
         assert res["res"][0] == [("str", 0, 1, 0.0, 0.1, True)]
+
+
+def test_spark_udf_return_nullable_array_field(spark):
+    class TestModel(PythonModel):
+        def predict(self, context, model_input):
+            values = [np.array([1, 2])] * (len(model_input) - 2) + [None, np.nan]
```

Comment:
```suggestion
            values = [np.array([1, np.nan])] * (len(model_input) - 2) + [None, np.nan]
```

what would happen if we run this code?

Reply:
works. , the np.nan inside the array is also converted to None.